docs/deploy/self_hosted_github_runners.md

```diff
@@ -0,0 +1,106 @@
+# Self Hosted GitHub Runners
+
+A self-hosted runner is a system that you deploy and manage to execute jobs from GitHub Actions on GitHub.com.
+This can be useful for
+
+* running automation scripts
+* deploying production / pre-production servers
+
+You will need to be an adminstrator to configure a self-hosted runner
+
+## Create Self-Hosted Runner
+
+You can add an actions runner from the settings page (change the organization/project information)
+
+```
+https://github.com/eclipse-pass/pass-docker/settings/actions/runners
+```
+
+Please refer to the latest from the settings page, but here is a (slightly modified) copy of the
+scripts.
+
+### Download the tools
+
+First we need to download the latest release from GitHub
```

Comment:
A link or reference to where you might find the latest actions-runner release would be helpful here.

Reply:
I re-worded to

```
Download the actions-runner code from GitHub, please
refer to `/settings/actions/runners/new` for the latest URLs and checksum value.
``

As this might be a fork of the project we don't really know the full URL so I left it as a relative `/settings/actions/runners/new` URL.  Please also see above that a screenshot was given with the GitHub look and feel for the code.